src/WriteLockStrategy/MariaDbMetadataLockStrategy.php

```diff
@@ -55,7 +55,7 @@ public function getLock(string $name): bool
         }
 
         $lockStatus = $res->fetchAll();
-        if ('1' === $lockStatus[0]['get_lock']) {
+        if ('1' === $lockStatus[0]['get_lock'] || 1 === $lockStatus[0]['get_lock']) {
```

Comment:
Fixes failing test and aligns with `\Prooph\EventStore\Pdo\WriteLockStrategy\MysqlMetadataLockStrategy`